docs/extensions/overview.md

```diff
@@ -137,10 +137,10 @@ Based on the resources discussed in GEP-1 the following table provides informati
 
 | Status             | Resource                | Providers                                                                            |
 |--------------------|-------------------------|--------------------------------------------------------------------------------------|
-| :white_check_mark: | `DNS`                   | [alicloud-dns<br>aws-route53<br>azure-dns<br>google-clouddns<br> openstack-designate](https://github.com/gardener/external-dns-management) |
-| :white_check_mark: | `OperatingSystemConfig` | [coreos](https://github.com/gardener/gardener-extension-os-coreos),[coreos-alicloud](https://github.com/gardener/gardener-extension-os-coreos-alicloud),[suse-jeos](https://github.com/gardener/gardener-extension-os-suse-jeos) |
-| :white_check_mark: | `BackupInfrastructure`  | alicloud<br>aws<br>azure<br>gcp<br>openstack                                         |
-| :white_check_mark: | `ControlPlane`          | alicloud<br>aws<br>azure<br>gcp<br>openstack<br>local (discontinued/replaced)        |
-| :white_check_mark: | `Infrastructure`        | [alicloud](https://github.com/gardener/gardener-extension-provider-alicloud)<br>[aws](https://github.com/gardener/gardener-extension-provider-aws)<br>[azure](https://github.com/gardener/gardener-extension-provider-azure)<br>[gcp](https://github.com/gardener/gardener-extension-provider-gcp)<br>[openstack](https://github.com/gardener/gardener-extension-provider-openstack)<br>local (discontinued/replaced) |
-| :white_check_mark: | `Worker`                | [alicloud](https://github.com/gardener/gardener-extension-provider-alicloud)<br>[aws](https://github.com/gardener/gardener-extension-provider-aws)<br>[azure](https://github.com/gardener/gardener-extension-provider-azure)<br>[gcp](https://github.com/gardener/gardener-extension-provider-gcp)<br>[openstack](https://github.com/gardener/gardener-extension-provider-openstack)<br>local (discontinued/replaced) |
-| :white_check_mark: | `Extension`             | [certificate-service](https://github.com/gardener/gardener-extension-shoot-cert-service) |
+| :white_check_mark: | `DNS`                   | [alicloud-dns<br>aws-route53<br>azure-dns<br>google-clouddns<br> openstack-designate<br> Cloudflare DNS](https://github.com/gardener/external-dns-management) |
```

Comment:
We have already the same info on https://github.com/gardener/gardener/tree/master/extensions#known-extension-implementations.
I guess we can put just a ref to https://github.com/gardener/gardener/tree/master/extensions#known-extension-implementations to avoid double maintenance.

Reply:
+1